lib/memjs/memjs.js

```diff
@@ -927,31 +927,28 @@ Client.prototype.quit = function() {
   }
 };
 
-// VERSION
-//
-// Request the server version from the "first" server in the backend pool
-//
-// The server responds with a packet containing the version string in the body with the following format: "x.y.z"
-Client.prototype.version = function(callback) {
+Client.prototype._version = function(serverKey, callback) {
```

Comment:
to follow the pattern of other commands, this should just take a server, not a serverkey